guides/parser_cache.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: guide
+doc_stub: false
+search: true
+title: GraphQL Parser cache
+section: Other
+desc: How to make parsing GraphQL faster with caching
+---
+
+Parser caching may be optionally enabled by setting `config.graphql.parser_cache = true` in your Rails application. The cache may be manually built by assigning `GrpahQL::Language::Parser.cache = GraphQL::Language::Cache.new("some_dir")`. This will create a directory (`tmp/cache/graphql` by default) that stores a cache of parsed files.
```

Comment:
```suggestion
Parser caching may be optionally enabled by setting `config.graphql.parser_cache = true` in your Rails application. The cache may be manually built by assigning `GraphQL::Language::Parser.cache = GraphQL::Language::Cache.new("some_dir")`. This will create a directory (`tmp/cache/graphql` by default) that stores a cache of parsed files.
```